src/framework/parsers/texture/basis.js

```diff
@@ -46,6 +46,7 @@ class BasisParser extends TextureParser {
 
     // our async transcode call provides the neat structure we need to create the texture instance
     open(url, data, device, textureOptions = {}) {
+        const format = textureOptions.srgb ? pixelFormatLinearToGamma(data.format) : data.format;
```

Comment:
It probably makes sense to do this in `Texture` constructor rather than every texture parser (looks like you missed `Img`, for example).

Reply:
`Img` handles it too, but as it only supported a single format, it's handled differently.

I considered handling it in Texture constructor, but did't like that the user passes in `format`, and would also pass in `srgb` flag, and under the hood we'd swap the format to a different format. I think code like this belongs to where it is now.